pom.xml

```diff
@@ -181,6 +181,11 @@
         <activeMQVersion>5.8.0</activeMQVersion>
         <commonsLangVersion>2.6</commonsLangVersion>
 
+        <!-- Elasticsearch -->
+        <elasticsearchMavenPluginVersion>2.0</elasticsearchMavenPluginVersion>
+        <!-- Keep in sync with the dependency required by Elasticsearch -->
+        <elasticsearchGroovyVersion>2.4.4</elasticsearchGroovyVersion>
```

Comment:
+1 for moving this to the root POM: There is an ES integration test in the JGroups module, too, which should be using the same version.